libvast/src/error.cpp

```diff
@@ -72,6 +73,9 @@ std::string render(caf::error err) {
   using caf::atom_uint;
   std::ostringstream oss;
   auto category = err.category();
+  if (atom_uint(category) == atom_uint("vast")
+      && static_cast<vast::ec>(err.code()) == ec::silent)
```

Comment:
```suggestion
      && err == ec::silent)
```

I haven't tested this, but I think this is possible.